code/datums/martial/buster_style.dm

```diff
@@ -44,6 +44,11 @@
 /datum/martial_art/buster_style/can_use(mob/living/carbon/human/H)
 	var/obj/item/bodypart/r_arm/robot/buster/R = H.get_bodypart(BODY_ZONE_R_ARM)
 	var/obj/item/bodypart/l_arm/robot/buster/L = H.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/l_arm/robot/buster/left
+	var/obj/item/bodypart/r_arm/robot/buster/right 
+	if(!((right && left) in H.bodyparts))
+		src.remove(H)
+		return
```

Comment:
move this down and do a !L && !R check or make an istype check here instead of doing this